react/Savings.tsx

```diff
@@ -27,7 +31,9 @@ const Savings: StorefrontFC<BasicPriceProps> = props => {
   }
 
   const previousPriceValue = commercialOffer.ListPrice
-  const newPriceValue = commercialOffer.Price
+  const spotPriceValue = commercialOffer.spotPrice ?? 0
+	const newPriceValue = spotPriceValue < commercialOffer.Price && usingSpotPrice ? spotPriceValue : commercialOffer.Price
```

Comment:
I think that this could have some undesirable side effects. What do you think about removing the `usingSpotPrice` prop and create new interpolable variables, such as `spotPrice`, `spotPriceSavingsValue` and `spotPriceSavingsPercentage`? This way, the client would have even more power and flexibility customizing the message!
Also, keep in mind that once  you do this, it will also be necessary to change the message files to reflect this new addition!

Reply:
@iaronaraujo Great idea! 'o' Ok then, I will do this changes.

Reply:
@iaronaraujo I've done this ;)